### PR TITLE
WEB-Microsoft-ofice365-csp1-SK1379

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ Voici la liste des logiciels compris dans la suite:
...
 
-- Être connecté à l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr)
+- Être connecté à l'[espace client OVHcloud](/links/manager)
 
 ## En pratique
 
 ### Commander une licence
 
-Rendez-vous sur notre site [OVHcloud.com](https://www.ovh.com/fr/office-365-business){.external} et sélectionnez la rubrique « Outils collaboratifs » de notre section « Web Cloud ». 
+Rendez-vous sur la page commercial [Microsoft 365 OVHcloud](https://www.ovh.com/fr/office-365-business){.external} et sélectionnez la rubrique `Microsoft`{.action} de notre section `Web Cloud`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md]:
```suggestion
Rendez-vous sur la page commerciale [Microsoft 365 OVHcloud](https://www.ovh.com/fr/office-365-business){.external} et sélectionnez la rubrique `Microsoft`{.action} de notre section `Web Cloud`{.action}.
```

**Review by Kazarus4 — APPROVED** [review]

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md [hunk 2]
```diff
@@ -47,14 +47,14 @@ Saisissez les informations relatives à l'utilisateur de la licence, cliquez sur
 
 ![emails](images/Outlook-cps1-02.png){.thumbnail}
 
-### Installer la suite Microsoft 365 sur votre machine
+### Installer la suite Microsoft 365 sur votre machine <a name="install360"></a>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md]: why a 360 anchor and not a 365?

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md [hunk 3]
```diff
@@ -65,14 +65,15 @@ Rendez-vous sur <https://portal.office.com/> et connectez-vous avec **l'adresse
 
 ![emails](images/Outlook-cps1-04.png){.thumbnail}
 
-> [!primary]
->
-> L'utilisateur peut installer la suite Microsoft 365 sur 5 machines sur Windows ou macOS, sur 5 tablettes et 5 smartphones.
->
+#### Installer la suite 365 sur plusieurs machines
+
+Avec une licence, l'utilisateur peut installer la suite Microsoft 365 sur **5 machines Windows et macOS**, sur **5 tablettes** et **5 smartphones**. Cela représente un total de **15 appareils** pour une licence. L'ensemble des ses appareils doivent appartenir au détenteur de la licence.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md]:
```suggestion
Avec une licence, l'utilisateur peut installer la suite Microsoft 365 sur **5 machines Windows et macOS**, sur **5 tablettes** et **5 smartphones**. Cela représente un total de **15 appareils** pour une licence. L'ensemble des ces appareils doivent appartenir au détenteur de la licence.
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md [hunk 4]
```diff
@@ -65,14 +65,15 @@ Rendez-vous sur <https://portal.office.com/> et connectez-vous avec **l'adresse
...
+
+Pour installer votre licence sur plusieurs machines, il vous faut simplement reproduire [l'étape précédente](#install360) *sur chaque appareils* en s'assurant que chacun est géré *par le même utilisateur*.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md]:
```suggestion
Pour installer votre licence sur plusieurs machines, il vous faut simplement reproduire [l'étape précédente](#install360) *sur chaque appareil* en vous assurant que chacun est géré *par le même utilisateur*.
```

Comment (Kazarus4) [reply pages/web_cloud/email_and_collaborative_solutions/microsoft_office/office_csp1/guide.fr-fr.md]: (#install360) => install365 ?

**Comment by BrenKt** [comment]
Translation to do from my side now

**Comment by BrenKt** [comment]
Translations check to do

**Pull request merged**